src/SourceTerm/PointSource.cpp

```diff
@@ -94,9 +94,9 @@ void seissol::sourceterm::transformMomentTensor(
     }
   }
 
-#if NUMBER_OF_QUANTITIES < 6
-#error You cannot use PointSource with less than 6 quantities.
-#endif
+// #if NUMBER_OF_QUANTITIES < 6
```

Comment:
Might need some generalization to the material class (in C++).

But I think it's good to leave that one out for now.